libs/wingsdk/src/std/json.ts

```diff
@@ -32,6 +32,32 @@ export class Json {
     throw new Error("Macro");
   }
 
+  /**
+   * Returns the values from the Json.
+   *
+   * @macro (Object.values($args$))
+   *
+   * @param json to get values from
+   * @returns the values from the Json as array of Json
+   */
+  public static values(json: Json): Json[] {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Returns the entries from the Json.
+   *
+   * @macro (Object.entries($args$))
+   *
+   * @param json to get values from
+   * @returns the entries from the Json as array of Json
```

Comment:
Maybe be specific about the [[key, value]] format here

Reply:
Updated the JSDoc.